testsuite/replay_verify.py

```diff
@@ -16,25 +16,26 @@
 
 
 TESTNET_RANGES = [
-    (962_000_000, 978_000_000),
-    (978_000_000, 994_000_000),
-    (994_000_000, 1_009_000_000),
-    (1_009_000_000, 1_025_000_000),
-    (1_025_000_000, 1_041_000_000),
-    (1_041_000_000, 1_057_000_000),
-    (1_057_000_000, 1_073_000_000),
-    (1_073_000_000, 1_088_000_000),
-    (1_088_000_000, 1_104_000_000),
-    (1_104_000_000, 1_120_000_000),
-    (1_120_000_000, 1_136_000_000),
-    (1_136_000_000, 1_151_000_000),
-    (1_151_000_000, 1_167_000_000),
+    (942_000_000, 958_000_000),
+    (958_000_000, 974_000_000),
+    (974_000_000, 990_000_000),
+    (990_000_000, 1_006_000_000),
+    (1_006_000_000, 1_022_000_000),
+    (1_022_000_000, 1_038_000_000),
+    (1_038_000_000, 1_054_000_000),
+    (1_054_000_000, 1_070_000_000),
+    (1_070_000_000, 1_086_000_000),
+    (1_086_000_000, 1_102_000_000),
+    (1_102_000_000, 1_115_000_000),
+    (1_115_000_000, 1_128_000_000),
+    (1_128_000_000, 1_141_000_000),
+    (1_141_000_000, 1_154_000_000),
+    (1_154_000_000, 1_167_000_000),
     (1_167_000_000, 1_183_000_000),
     (1_183_000_000, 1_199_000_000),
     (1_199_000_000, 1_215_000_000),
     (1_215_000_000, 1_230_000_000),
     (1_230_000_000, 1_246_000_000),
-    (1_246_000_000, 1_262_000_000),
```

Comment:
we probably dont wanna remove this shard, just skip tapos transactinos (starts roughly around 1259000000)

Reply:
I ran another workflow to test setting the end of the range https://github.com/aptos-labs/aptos-core/actions/runs/9714540293.

For this PR I just deleted the last range and added another one at the beginning. I also reduced the size for some of the longer running ranges